Hamilton_Matthew/app_base.htm

```diff
@@ -0,0 +1,93 @@
+<%inherit file="/catalog/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="nav">
+    <li class="${'activeaf' if request.dmp.app == "homepage" and request.dmp.page == 'index' else ''}"><a href="/index/">Home</a></li>
+    <li class="${'activeaf' if request.dmp.page == 'about' else ''}"><a class="nav-link" href="/about/">About Us</a></li>
+    <li class="${'activeaf' if request.dmp.page == 'contact' else ''}"><a class="nav-link" href="/contact/">Contact</a></li>
+    <li class="${'activeaf' if request.dmp.page == 'terms' else ''}"><a class="nav-link" href="/terms/">Terms</a></li>
+    <li class="${'activeaf' if request.dmp.page == 'faq' else ''}"><a class="nav-link" href="/faq/">FAQ</a></li>
+    <li class="${'activeaf' if request.dmp.app == 'catalog' and request.dmp.page == 'index' else ''}"><a class="nav-link" href="/catalog/index">Products</a></li>
```

Comment:
Very wise way to keep the navbar consistent across the site while keeping the current tab set to 'active'!